clients/drcachesim/tracer/instr_counter.cpp

```diff
@@ -72,28 +80,98 @@ static uint64 instr_count;
 // but to avoid the complexity of different instrumentation we need a threshold.
 #define DELAY_FOREVER_THRESHOLD (1024 * 1024 * 1024)
 
-static std::atomic<bool> reached_trace_after_instrs;
+static std::atomic<bool> reached_trace_after_instrs(false);
 std::atomic<uint64> retrace_start_timestamp;
 
+static std::atomic<uint> window_idx(0);
```

Comment:
Maybe change this identifier and add a comment to make it clearer that this applies only to irregular window tracing. Otherwise it may be confused with the current trace window as it applies to other trace window features.

Reply:
Done.
Renamed to `irregular_window_idx` (also renamed `num_irregular_windows` variable).